docs/site/tutorials/todo-list/todo-list-tutorial-relations.md

```diff
@@ -48,9 +56,9 @@ to move to the next step to create the controller.
 
 #### Relation decorators
 
-When specifying the `hasMany` relation, it adds `@hasMany()` decorator defines
-this property. As the decorator's name suggests, `@hasMany()` informs LoopBack 4
-that a todo list can have many todo items.
+When we added the `hasMany` relation using the `lb4 relation` command, it added
```

Comment:
> When we added the `hasMany` relation using the `lb4 relation` command, it added
`@hasMany()` decorator defines this property

When we added the `hasMany` relation using the `lb4 relation` command, it added `the`
`@hasMany()` decorator  `which` defines this property . (Is this what we want to say?)

Reply:
actually, it added the property + the decorator. How about: 

When we added the `hasMany` relation using the `lb4 relation` command, it added the
`@hasMany()` decorator together with this property.

Reply:
> actually, it added the property + the decorator. How about:
> 
> When we added the `hasMany` relation using the `lb4 relation` command, it added the
> `@hasMany()` decorator together with the `todo` property.

`todo` or `todos` ?  The line below says:

![image](https://user-images.githubusercontent.com/6864736/68954118-5a02b280-0791-11ea-9f43-671c1fb36dbf.png)

Reply:
Also, we still need a `the` before '@hasMany decorator'

Reply:
it should be `todos` property (the one in the documentation is correct). 

OK. how about this:

When we added the `hasMany` relation using the `lb4 relation` command, it added
the `@hasMany()` decorator together with the `todos` property.